blocks/hero/hero.css

```diff
@@ -10,6 +10,23 @@ main .hero-container + .section {
   padding-block-start: var(--default-content-gap);
 }
 
+.section.hero-container {
+  background-color: var(--clr-blue)
+}
+
+.two-columns .section.hero-container {
+  width: 100vw;
+  position: relative;
+  left: 50%;
+  right: 50%;
+  margin-left: -50vw;
+  margin-right: -50vw;
+}
+
+.section.hero-container.hero-container.alternate-hero {
+  background-color: var(--clr-coral)
+}
+
 main .hero-container.alternate-hero {
   --hero-bg-color: var(--clr-coral);
```

Comment:
@seanmilfort, since we are setting the background on a wrapper parent container ".section.hero-container", do we still need to also add the background on "main .hero .hero-content-container"?

Can you test removing the background property on "main .hero .hero-content-container" and see if it still works?